### DISCUSSION TITLE
Floem should have a roadmap

### DISCUSSION THREAD

**Discussion by golanguage** [lapce/floem#462] [General]
So the users and potential users can know what the project has achieved, what the project want to do.
And know  the Floem will be a  independent project for general , or a attachment of Lapce.

iced have this one:
https://whimsical.com/roadmap-iced-7vhq6R35Lp3TmYH4WeYwLM

seem very useful.

**Comment by bowenxuuu** [comment]
Yes !

**Comment by jrmoulton** [comment]
This is in the works. I've been creating issues for important fixes and features and I'll soon collect them into a milestone or project somehow. But for now, the issues contain items that are considered "in the roadmap"
